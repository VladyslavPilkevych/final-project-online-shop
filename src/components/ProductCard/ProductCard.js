import React, { memo } from 'react';

function ProductCard() {
  return (
    <section>
      <h1>Product Card</h1>
    </section>
  );
}

export default memo(ProductCard);
