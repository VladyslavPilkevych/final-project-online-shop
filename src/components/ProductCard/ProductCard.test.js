import { render } from '@testing-library/react';
import ProductCard from './ProductCard';

describe('Snapshot ProductCard should render', () => {
    test('Snapshot ProductCard is rendered', () => {
      const { asFragment } = render(<ProductCard />);
      expect(asFragment()).toMatchSnapshot();
    });
  });