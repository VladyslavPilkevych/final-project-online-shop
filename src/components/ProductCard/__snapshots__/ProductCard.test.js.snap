// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot ProductCard should render Snapshot ProductCard is rendered 1`] = `
<DocumentFragment>
  <section>
    <h1>
      Product Card
    </h1>
  </section>
</DocumentFragment>
`;
